samples/command-line-terminal.yaml

```diff
@@ -0,0 +1,31 @@
+# It's just an example of workspace configuration OpenShift Console creates
+# when user requests terminal
+apiVersion: workspace.che.eclipse.org/v1alpha1
```

Comment:
just a question about the version here... v1alpha1? why not something simpler like v0.0.1? on line 17 you use apiVersion 1.0.0 so I'm just a bit confused how the apis are versioned.

Reply:
I did not have enough time to learn much enough how CRD is supposed to be designed, so my simple answer is: 
- apiVersion of k8s object consist of apiGroup - workspace.che.eclipse.org + version v1alpha1;
- Our Workspace CRD is not stable enough to move it from alpha version I think.

I think K8s has conversion about versions for apps, and versions are not supposed to be MAJOR.MINOR.PATCH. It should be `vXalphaX`, `vXbetaX`, `vX`.

@amisevsk Do you have something to add?

Reply:
Yeah, kubernetes API version conventions dictate this sort of naming: https://kubernetes.io/docs/concepts/overview/kubernetes-api/#api-versioning